src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -309,7 +309,7 @@ public void Track(ITelemetry telemetry)
                 // invokes the Process in the first processor in the chain
                 this.configuration.TelemetryProcessorChain.Process(telemetry);
 
-#if NET46
+#if NET46 || NET45
```

Comment:
EventSource is available from 4.5. Is 4.0 important? I may want to look at .Net Core next.